packages/tanstack-solid-query/test/polymorphism.test.tsx

```
@@ -0,0 +1,324 @@
+          );
+        });
+
+        await userEvent.click($.getByTestId('refreshBtn'));
+
+        await vi.waitFor(() => {
+          expect($.container).toHaveTextContent(
+            'Last Export: `Search for Polymorphism Solid` (Ready!)',
+          );
+        });
+      } finally {
+        await ctx[Symbol.asyncDispose]();
```

ESLint

[error] Usage of Symbol.asyncDispose is not allowed - use `makeAsyncResource()`

(no-restricted-syntax)